modules/installation-aws-permissions.adoc

```diff
@@ -152,6 +152,11 @@ If you use an existing Virtual Private Cloud (VPC), your account does not requir
 * `iam:TagInstanceProfile`
 * `iam:TagRole`
 
+[NOTE]
+=====
+If you specify an existing IAM role in the `install-config.yaml` file, the following IAM permissions are not required: `iam:CreateRole`,`iam:DeleteRole`, `iam:DeleteRolePolicy`, and `iam:PutRolePolicy`.
```

Comment:
I'm fine with this as-is, but the roles might read better if they're in a bulleted list. I'll leave that up to you. :slightly_smiling_face: